zookeeper-docs/src/main/resources/markdown/zookeeperTools.md

```diff
@@ -425,6 +426,38 @@ All layers compared.
 
 Or use `^c` to exit interactive mode anytime.
 
+
+<a name="zkWatchBenchmark"></a>
+
+### zkWatchBenchmark.sh
+- A benchmark tool that benchmarks the watch throughput and latency, which supports multiple clients threads, multiple watch modes
+- See **ZOOKEEPER-3823** for the design document
+- Notes:
+    - `export JVMFLAGS="-Xms12g -Xmx12g"` to have more JVM heap size to avoid `GC overhead limit exceeded` when you have a large scale testing
+    - `-v` to print a detailed logs when you find no benchmark output result for a long time
+- Usages:
+
+
+        nohup ./zkWatchBenchmark.sh -root_path /bench-watch -threads 100 -znode_count 1000 -force -connect_string \n
```

Comment:
I think nohup is unnecessary to include in the example command (while useful here I agree, there are other options, and everyone can decide how to handle the long runtime - at least this is how I see it, not a blocker though)